aten/src/ATen/core/TensorTypeId.cpp

```diff
@@ -1,5 +1,9 @@
 #include "ATen/core/TensorTypeId.h"
 
+namespace at {
```

Comment:
related to the other comment, why don't we do the opposite and move this to Formatting?  Is it because there are external usages that break?

Reply:
I think it's silly to have to include `Formatting.h` if you want to `<<` types; the implementations are really simple, put them with the actual headers.